### ISSUE TITLE
How are the learned prototypes used when inferencing unseen pairs?

### ISSUE THREAD

**Issue by wqshmzh** [quhongyu/ClusPro#6]
In "val_forward" function, I noticed that the tensors "featattr_memory_con" and "featobj_memory_con" are not used in the final logits. The attribute, object, and pair final logits are the cosine similarities between the CLIP-encoded image vectors and all CLIP-encoded text vectors. The paper only introduced the inferencing as "During testing, the test image x is fed into CLUSPRO to obtain prediction scores for attribute p(ai ∣x), object p(oi ∣x), and composition prediction p(ci,j ∣x)."  However, descriptions of p(ai|x), p(oi|x) and p(ci,j|x) are shown only in section 3.2 "Baseline Architecture". How are the three probabilities computed in CLUSPRO?  How are the two kinds of prototypes used when inferencing?
